app/views/contexts/summary/group.py

```diff
@@ -77,10 +93,48 @@ def _build_blocks(
                     ]
                 )
 
+            elif block["type"] == "ListCollector":
+                section = schema.get_section(location.section_id)
+
+                if summary_item := schema.get_summary_item_for_list_for_section(
+                    section_id=section.get("id"),
+                    list_name=block.get("for_list", {}),
```

Comment:
These are mandatory for list collector blocks.
```suggestion
                    section_id=section["id"],
                    list_name=block["for_list"],
```